conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conans import ConanFile, CMake, tools
+
+
+class AudresampleLibConan(ConanFile):
+    name = "audresamplelib"
+    version = "latest"
+    license = "Proprietary"
```

Comment:
I could not find a license file. Shouldn't we also make this licensed under MIT as auresample?

Reply:
As far as I understood this library, we just use [soxr](https://sourceforge.net/projects/soxr/) as the main ingredient. It is published under LGPLv2. If this is compatible with using MIT here, I would be in favor of switching to MIT.

Reply:
I don't think they are compatible.

Reply:
Then there would be a license conflict with audresample anyway and we would need to change it.

Reply:
See also from [Wikipedia](https://en.wikipedia.org/wiki/GNU_Lesser_General_Public_License):

> Paragraph 5 of the LGPL version 2.1 states:
>
>A program that contains no derivative of any portion of the Library, but is designed to work with the Library by being compiled or linked with it, is called a "work that uses the Library". Such a work, in isolation, is not a derivative work of the Library, and therefore falls outside the scope of this License.

Reply:
Ah yes. Seems that was the important difference between LGPL and GPL. Where do we need to a reference to the original code?

Reply:
I think inside https://github.com/audeering/audresample as there we ship only the binary.
To some degree we do this already: 
![image](https://user-images.githubusercontent.com/173624/204740915-ac02aeea-b925-4938-9c0a-b2f8c3763ea5.png)

Reply:
So in this project the link in the README is enough, right? Then I would just also add the MIT here and set it in the conanfile.

Reply:
Yes, sounds sufficient.

Reply:
Ok, done.